---
layout: post
title:  "HZAU-Bioinformatics-Collection"
date:   2019-06-13 13:31:01 +0800
categories: Backup
tag: Backup
---

* content
{:toc}



Chapter 1			{#backup}
------------------------


TEST


Chapter 2    {#backup} 
-------------------------------

Chapter 5 Gene Analysis and Annotation    {#backup} 
-------------------------------
## Gene Annotation
### prokaryotic & mRNA
+ [orffinder](https://www.ncbi.nlm.nih.gov/orffinder/){:target="_blank"}
+ [FGENESB](http://linux1.softberry.com/berry.phtml?topic=fgenesb&group=programs&subgroup=gfindb){:target="_blank"}
+ [GeneMarkS](http://exon.biology.gatech.edu/GeneMark/genemarks.cgi){:target="_blank"}

### Eukaryotic
+ [GenScan](http://hollywood.mit.edu/GENSCAN.html)
+ [GeneMark](http://exon.biology.gatech.edu/GeneMark/gmhmme.cgi)
+ [Augustus](http://bioinf.uni-greifswald.de/augustus/submission.php)

### Comparative Genomics 
+ [N-SCAN/Twinscan](http://mblab.wustl.edu/nscan/)

## Gene Sturcture
### Gene Features
+ [BCM](http://arete.ibb.waw.pl/PL/html/gene_feature_searches_bcm.html)
### Promoter
+ [Promoter 2.0](http://www.cbs.dtu.dk/services/Promoter/)
+ [PLACE(plant)](https://www.dna.affrc.go.jp/PLACE/?action=newplace)
+ [JASPAR](http://jaspar.genereg.net/inference)
+ [PlantPAN](http://plantpan.itps.ncku.edu.tw/promoter.php)
+ [Matlnspector](https://www.genomatix.de/online_help/help_matinspector/matinspector_help.html)
+ [TRANSFAC](http://genexplain.com/transfac/)
+ [PROMO](http://alggen.lsi.upc.es/cgibin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3)
+ [Tfsitescan](http://www.ifti.org/cgi-bin/ifti/Tfsitescan.pl)
### lncRNA
+ [PLEK](https://sourceforge.net/projects/plek/)
+ [CNCI (Coding-Non-Coding Index)] (https://github.com/www-bioinfo-org/CNCI)
+ [CPAT](http://lilab.research.bcm.edu/cpat/)
+ [NONCODE](http://www.noncode.org/blast.php)
+ [LNCipedia](https://lncipedia.org/)
+ [NRED](https://www.hsls.pitt.edu/obrc/index.php?page=URL1237993821)
+ [PLncDB](https://omictools.com/plncdb-tool)
### microRNA Animal
+ [miRBase](http://www.mirbase.org/search.shtml)
+ [miRDB](http://mirdb.org/miRDB/mining.html)
+ [miRecords](http://mirecords.umn.edu/miRecords/)
### microRNA Bacteria
+ [TargetRNA2](http://cs.wellesley.edu/~btjaden/TargetRNA2/)
### microRNA Plant
+ [WMD3](http://wmd3.weigelworld.org/cgi-bin/webapp.cgi)
## ENCODE: Encyclopedia of DNA elements
+ [fruitENCODE](http://www.epigenome.cuhk.edu.hk/encode.html)


Chapter 6 Protein
-----------------
## Biophysics analysis
+ [ExPASy](https://www.expasy.org/)
+ [CBS](http://www.cbs.dtu.dk/services/)
+ [ANTHEPROT](http://antheprot-pbil.ibcp.fr/)
+ [DNAMAN](http://www.lynnon.com/)

## Motif
+ [prosite](https://prosite.expasy.org/)
+ [NCBI Conserved Domains](https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi)
+ [SMART](http://smart.embl-heidelberg.de/)
+ [PredictProtein](https://www.predictprotein.org/)

## Second Structure
+ [ANTHEPROT] (https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_sopma.html)
+ [COILS] (https://embnet.vital-it.ch/software/COILS_form.html
NP_176610)

## Third Structure
+ [Phyre2] http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index 直接输入蛋白质序列和E-mail，2小时后反馈结果，同 时给出与之同源的蛋白质的3D结构。
+ [SWISS-MODEL Workspace](https://swissmodel.expasy.org/interactive) 提供Automated Mode、Alignment Mode、Project Mode 等多种分子建模方式

##  membrane Protein

+ [DAS：分析原核生物蛋白质的跨膜结构] (http://www.sbc.su.se/~miklos/DAS/maindas.html） +  [TMpred：分析蛋白质的跨膜结构和在膜上的方向] (http://www.ch.embnet.org/software/TMPRED_form.html） 
+ [HMMTOP: 分析蛋白质跨膜结构和方向] (http://www.enzim.hu/hmmtop/html/submit.html) 
+ [TopPred: 分析细胞膜蛋白拓扑结构] (http://mobyle.pasteur.fr/cgi-bin/portal.py?#forms::toppred) 
+ [TMHMM: 分析蛋白质跨膜结构] (http://www.cbs.dtu.dk/services/TMHMM/)
+ [TMHMM](http://www.cbs.dtu.dk/services/TMHMM/)
+ [MEMSAT3](http://bioinf.cs.ucl.ac.uk/software_downloads/memsat/)
+ [HMMTOP](http://www.enzim.hu/hmmtop/index.php)
+ [TMpred](http://www.ch.embnet.org/software/TMPRED_form.html)
+ [DAS](http://www.sbc.su.se/~miklos/DAS)
+ [TOPCONS](http://topcons.cbr.su.se/)
+ [PRED-TMR](http://athina.biol.uoa.gr/PRED-TMR/)
+ [TopPred1.10](http://mobile.Pasteur.fr/cgi-bin/portal.py?#forms::toppred)
+ [CCTOP](http://cctop.enzim.ttk.mta.hu/)
+ [PRED-TMBB](http://bioinformatics.biol.uoa.gr/PRED-TMBB/)
+ [GPI-SOM](http://pgi.unibe.ch/)

## Secretion Signals 
+ [SignalP](http://www.cbs.dtu.dk/services/SignalP/)
+ [Phobius](http://phobius.cgb.ki.se)
+ [Philius](http://www.yeastrc.org/philius)
+ [Spoctopus](http://octopus.cbr.su.se)
+ [Signal-3L](http://www.csbio.sjtu.edu.cn/bioinf/Signal-3L)
+ [MEMSAT-SVM](http://bioinf.cs.ucl.ac.uk/psipred/?memsatsvm=1)
+ [LipoP](http://www.cbs.dtu.dk/services/LipoP/)
+ [SecretomeP](http://www.cbs.dtu.dk/services/SceretomeP/)
+ [TatP](http://www.cbs.dtu.dk/services/TatP-1.0/)
+ [Signal-3L: A 3-layer approach for predicting signal peptides ](http://www.csbio.sjtu.edu.cn/bioinf/Signal-3L/)
+ [PrediSi: PREDIction of SIgnal peptides ](http://www.predisi.de/)
+ [Phobius: A combined transmembrane topology and signal peptide predictor ](http://phobius.sbc.su.se/)
+ [PEPLip: Predictor of Signal Peptide and Lipoprotein Cleavage Sites in Proteins ](http://gpcr.biocomp.unibo.it/cgi/predictors/spep/pred_spepcgi.cgi）

## N-glc site
+ [NetOGly](http://www.cbs.dtu.dk/services/NetOGlyc/)
+ [其他预测工具](http://chemdata.shu.edu.cn/gal_p/)
+ [others01](http://comp.chem.nottingham.ac.uk/cgi-bin/glyco/bin/getparams.cgi)
+ [others02](http://bioinformatics.cau.edu.cn/zzd_lab/CKSAAP_OGlySite/index.html)
+ [others03](http://turing.cs.iastate.edu/EnsembleGly/predict.html)

## Protein Modification
+ [NetAcet: 乙酰化位点预测](http://www.cbs.dtu.dk/services/NetAcet/)
+ [NetPhos: 磷酸化位点预测](http://www.cbs.dtu.dk/services/NetPhos/)
+ [NMT: 肉豆蔻酰化位点预测化]( http://mendel.imp.ac.at/myristate/SUPLpredictor.htm)
+ [PACMAN: 甲基化预测](http://bugfri.vital-it.ch/)
+ [PrePS: 异戊二烯化预测](http://mendel.imp.ac.at/sat/PrePS/index.html)
+ [SUMOplot: SUMO化预测](http://www.abgent.com/sumoplot)

## Subcellular localization
+ [PSORT](https://wolfpsort.hgc.jp/)
+ [CELLO ](http://cello.life.nctu.edu.tw)
+ [Subnuclear Compartments Prediction System ](http://array.bioengr.uic.edu/subnuclear.htm )
+ [Proteome Analyst Specialized Subcellular Localization Server ](http://webdocs.cs.ualberta.ca/~bioinfo/PA/Sub)
+ [Lokero/TargetLoc/SherLoc2/MultiLoc2/YLoc ](http://abi.inf.uni-tuebingen.de/Research/Systems%20Biology/protein-subcellular-localization 
+ [TargetP ](http://www.cbs.dtu.dk/services/TargetP)
+ [Predotar ](https://urgi.versailles.inra.fr/predotar/predotar.html )
+ [Protein Prowler ](http://bioinf.scmb.uq.edu.au:8080/pprowler_webapp_1-2)
+ [PSORT ](http://www.psort.org)
+ [PredictProtein ](https://www.predictprotein.org LocTree ](https://rostlab.org/services/loctree3/)

![Simulation]({{ '/styles/images/nelb-fadd.png' | prepend: site.baseurl  }})
![Simulation2]({{'/styles/images/ksmh0-a3y4c.gif'| prepend: site.baseurl  }})

[MD simulations Refference](http://ibi.hzau.edu.cn/ComputationalBiochemistry/index2.html)

[test my website](https://xanadu314.github.io/mdsimulations/)
 

[jekyll]:      http://jekyllrb.com
[jekyll-gh]:   https://github.com/jekyll/jekyll
[jekyll-help]: https://github.com/jekyll/jekyll-help
